src/scene/index.js

```diff
@@ -141,19 +152,20 @@ function onMouseUp() {
   canvas.removeEventListener('mouseup', onMouseUp, false);
   canvas.addEventListener('mousemove', onMouseHoverMove, false);
   canvas.style.cursor = 'grab';
+
+  // Allow Drag Rotation "inertia" to continue for 500ms after dragging ends
+  setTimeout(() => {
+    drag = false;
+  }, 500);
 }
 
 function onMouseOver() {
   canvas.addEventListener('mousedown', onMouseDown, false);
   canvas.addEventListener('mousemove', onMouseHoverMove, false);
 }
 
-function onMouseHoverMove() {
-  const rect = renderer.domElement.getBoundingClientRect();
-  mouse2.x = ((event.clientX - rect.left) / (rect.right - rect.left)) * 2 - 1;
-  mouse2.y = -((event.clientY - rect.top) / (rect.bottom - rect.top)) * 2 + 1;
-
-  raycaster.setFromCamera(mouse2, camera);
+function onMouseHoverMove(event) {
```

Comment:
I actually have no idea how this function was working at all 🐒 

Before, it didn't have the `event` param defined, which is passed. Yet...it was using the undefined variable as an object below... 🤷 Fixed now.